articles/libraries/lock/v11/index.md

```diff
@@ -0,0 +1,142 @@
+
+::: panel Hosted Login Page
+Lock 11 is a version of Lock tailored for embedded login in your applications. If you use Auth0's [Hosted Login Page](/hosted-pages/login), Lock 11 is not necessary and does not work inside the Hosted Login Page. The Hosted Login Page remains the simplest and most secure method by which to authenticate users for your applications. Lock 11 is designed to make cross-origin authentication work smoothly for when you cannot use the Hosted Login Page and need to embed a login in your application.
+:::
+
+## Lock Installation
+
+You can install Lock 11 via several methods. Select any of the following installation sources that best suit your environment and application.
+
+### Installation Sources
+
+Install via [npm](https://npmjs.org):
+
+```sh
+npm install auth0-lock
+```
+
+Install via [bower](http://bower.io):
+
+```sh
+bower install auth0-lock
+```
+
+Include via our CDN (Replace `.x` and `.y` with the latest minor and patch release numbers from the [Lock Github repository](https://github.com/auth0/lock/releases):
+
+```html
+<!-- Latest minor release -->
+<script src="https://cdn.auth0.com/js/lock/11.x/lock.min.js"></script>
+
+<!-- Latest patch release (recommended for production) -->
+<script src="https://cdn.auth0.com/js/lock/11.x.y/lock.min.js"></script>
+```
+
+::: note
+It is recommended that production applications use a specific patch version, or at the very least a specific minor version. Regardless of the method by which Lock is included, the recommendation is that the version should be locked down and only manually updated, to ensure that those updates do not adversely affect your implementation. Check the [GitHub repository](https://github.com/auth0/lock/releases) for a current list of releases.
+:::
+
+### Mobile
+
+If you are targeting mobile audiences, Auth0 recommends that you add the following meta tag to your application's `head`:
+
+```html
+<meta name="viewport" content="width=device-width, initial-scale=1"/>
+```
+
+### Bundling Dependencies
+
+If you are using browserify or webpack to build your project and bundle its dependencies, after installing the `auth0-lock` module, you will need to bundle it with all its dependencies. Examples are available for [Browserify](https://github.com/auth0/lock/tree/master/examples/bundling/browserify) and [webpack](https://github.com/auth0/lock/tree/master/examples/bundling/webpack).
+
+### Cross-Origin Authentication
+
+Embedding Lock within your application, rather than using the [Hosted Login Page](/hosted-pages/login), requires [cross-origin authentication](/cross-origin-authentication). In order to use embedded Lock via cross-origin authentication, you ensure that the client settings area of the [Dashboard]($manage_url}), in the **Advanced Settings** menu, under the **OAuth** tab, ensure that the **OIDC Conformant** and **Cross Origin Authentication** settings are turned on.
```

Comment:
CO authentication works with both OIDC-Conformant and non-OIDC-Conformant apps, and the **Cross-Origin Authentication** toggle won't be present anymore.
We can note, however, that 
> Embedding Lock within your application requires [cross-origin authentication](/cross-origin-authentication), which is only enabled for [first-party clients](/link-to-relevant-doc).